### ISSUE TITLE
Support for uint64

### ISSUE THREAD

**Issue by tperkinson** [appnexus/pyrobuf#11]
Awesome package! 

Pyrobuf seems to have an issue with serializing (or deserializing, not sure which) u64int that is greater than 0x7FFFFFFF (31 bits) as seen in the following example.  the Json and Dict representations seem to be fine.

System: Mac OSX using 64 bit python and Cython from an integrated Anaconda install:

Example:
Using pyrobuf-0.0.2a4 with the following protobuf definition:

message A {
        optional  uint64 a0 = 1;
}

Iand the following code:
pb2 = a_proto.A()
pb2.a0 = 0x7FFFFFFF
print pb2.a0
print pb2.SerializeToJson()
print pb2.SerializeToDict()
print a_proto.A().FromString(pb2.SerializeToString())

print
pb = a_proto.A()
pb.a0 = 0x80000000
print pb.a0
print pb.SerializeToJson()
print pb.SerializeToDict()
print a_proto.A().FromString(pb.SerializeToString())

Produces the following result:
------------------------
2147483647
{"a0": 2147483647}
{'a0': 2147483647L}
a0: 2147483647

2147483648
{"a0": 2147483648}
{'a0': 2147483648L}
a0: 18446744071562067968

**Comment by 3kwa** [comment]
A quick glance makes me think it is a varint encoding/decoding problem in pyrobuf_util.pyx.

**Comment by 3kwa** [comment]
https://github.com/appnexus/pyrobuf/pull/12 should fix the problem!

**Comment by tperkinson** [comment]
I really appreciate the help.  I tried it and am still getting the same result.

I've rolled the version via pip, verified the right version via pip show pyrobuf.  Then I looked at pyrobuf_util.pyx in the src in anaconda/lib/python2.7/site-packages/pyrobuf and it seems to reflect the code change described.  I rolled to a new protobuf description to ensure a clean build.  All efforts result in the same described behavior.

Will continue to look at it.

**Comment by mmoresco** [comment]
I had to manually rm the old pyrobuf egg from site-packages, but then it worked for me.

**Comment by tperkinson** [comment]
Thank you for the pointer.  I cleaned EVERYTHING from my system but missed a wheel.  When I removed those, everything rebuilt and works!  THANK you both.

As an aside, the speed gained from  switching away from google protobuf is ENORMOUS.  it may not matter on some projects, but when dealing with millions of protobufs it makes all the difference in the world (and the serialization to other formats is a big help also).
